junifer/markers/temporal_snr/temporal_snr_base.py

```diff
@@ -0,0 +1,114 @@
+"""Provide abstract base class for temporal signal-to-noise ratio (tSNR)."""
+
+# Authors: Leonard Sasse <l.sasse@fz-juelich.de>
+# License: AGPL
+
+
+from abc import abstractmethod
+from typing import Any, Dict, List, Optional, Union
+
+from ...utils import raise_error
+from ..base import BaseMarker
+from ..utils import _voxelwise_tsnr
```

Comment:
why is this in utils? Isn't it only from this method?

Reply:
It seemed like the place to keep when the PR was started, now of course, it makes sense it in it's own submodule.

Reply:
Updated.